module/VuFind/src/VuFind/Auth/SIP2.php

```diff
@@ -117,9 +118,6 @@ public function authenticate($request)
         ) {
             // Success!!!
             $user = $this->processSIP2User($result, $username, $password);
-
-            // Set login cookie for 1 hour
-            $user->password = $password; // Need this for Metalib
```

Comment:
I don't know what the point of this was in the distant past, but I'm confident it no longer has any effect and should be deleted. Whether this authentication method works at all is a separate question; I doubt it has been used in the past decade or more.